ui/k-toolbar/main.kytos

```diff
@@ -5,6 +5,19 @@
                           icon="desktop"
                           :on_click="list_windows">
                 </k-button>
+      <k-accordion>
+        <k-accordion-item title = "Create Maintenance Window">
+                <k-input placeholder = "Id" :value.sync ="id"></k-input>
+                <k-input placeholder = "Description" :value.sync = "description"></k-input>
+                <k-input placeholder = "Start time" :value.sync = "start"></k-input>
+                <k-input placeholder = "End time" :value.sync = "end"></k-input>
+                <k-select icon = "link" title = "List of Switches" :options = "dpid_names" :value.sync = "items"></k-select>
+                <k-select icon = "link" title = "List of Interfaces" :options = "interface_names" :value.sync = "interface_items"></k-select>
+                <k-select icon = "link" title = "List of Links" :options = "link_names" :value.sync = "link_items"></k-select>
+                <k-input placeholder = "Status, Input 0, 1, or 2" :value.sync = "status" ></k-input>
+                <k-button title = "Post" :on_click = "post_window"></k-button>
+        </k-accordion-item>
```

Comment:
@rohanCodes001, could you put this new `k-accordion-item` to come before the button to list the windows? Just so the layout looks similar to other ones that we have, for instance check out this one on `mef_eline`:

![2022-03-10-104659_231x415_scrot](https://user-images.githubusercontent.com/1010796/157678741-5c83d53a-b63e-45d3-b925-dc703f5212c1.png)

Also, could you please [put the button to list the maintenance window](https://github.com/kytos-ng/maintenance/blob/master/ui/k-toolbar/main.kytos#L3-L7) in a new accordion item, this wasn't related to the scope of this PR, but since you're working on this component, I think it'd be worth addressing it